homeassistant/components/light/strings.json

```diff
@@ -308,7 +308,7 @@
         },
         "flash": {
           "name": "Flash",
-          "description": "If the light should flash."
+          "description": "Tell light to flash, can be either value short or long."
```

Comment:
@c0ffeeca7 what do you think?

Reply:
Oh now that is interesting, I was using this with the Developer Tools section, and if I remember correctly, it was simply a guide for what to enter in yaml mode.  I'm afraid I'm having internet difficulties at home so I can't get in through nabu casa to check, but I'll have a proper look when the internet goes back up there.

Reply:
Okay, I can confirm that in Developer Tools, on the Services tab, no such UI option exists.  Flash does not show up as an option at all on UI mode and in YAML mode, the following screenshot demonstrates what is shown: 
<img width="1278" alt="Screenshot 2023-08-14 at 10 52 49" src="https://github.com/home-assistant/core/assets/642320/c269fe0c-f83b-4692-a9e0-86c46931bb50">
Here the next, admittedly in my own opinion, suggests to me that the flash option is a boolean.  I hope this clarifies why I felt this was needed :)

Reply:
Even more curious, when I switched to using Turn Off, the UI did indeed show the option you've described.  Though my home internet went off line again so I can't test it, joy.  I am really starting to hate my ISP, I don't understand what they're doing of late and I'm locked in a contract with them till the end of the year.

Reply:
I think we should close this PR given @c0ffeeca7 's comment and that it makes sense as you see the full context.

Reply:
Hi @gjohansson-ST, I've little say in the game I admit, but I really do find myself disagreeing, as you only see the full context in situations where the GUI is being used.  For those of us who naturally use the yaml view, the documentation table shown on the developer tools service section is very much what we rely on, and as seen in the screenshot above, there is no context shown.  Matching up with what is listed on the documentation would add this clarity and I'm certain would not actually detract from the user experience of those using the graphical interface.  I also think matching up consistency between the two bits of documentation is just good HCI practice, but that's partly my honours professor from my university days' voice nagging me in my head, 11 years later and her voice is still back there nagging me! 😄

I do however agree that if the flash command is in fact present on the turn off, then I will adjust this PR and submit a new one for updating the website documentation there too.  I'm afraid I didn't manage to confirm when I got home last night as the little ones decided that bed time was going to take all night and I fell asleep next to a grumpy two year old instead 😅

Luckily, my ISP is behaving right now so I'll be able to test remotely shortly 😀 I just need to get a few bits of work done first but I'll check in at lunchtime (a little over two hours from now her in Scotland:) So please hold off on closing until at least end of day 😅

Reply:
I must confirm now that the UI option for flash does indeed show up for me.  I must have just been using a device that didn't support the feature.  I can also though confirm that the yaml mode documentation table still provides no context so I still think my change to the text holds value :)

Reply:
I see your point, @apbarratt. When I wrote my feedback, I hadn't considered the YAML-only view.